include/cantera/oneD/Flow1D.h

```diff
@@ -670,34 +675,211 @@ class Flow1D : public Domain1D
+        return 2.0*(A_right - A_left) / (z(j+1) - z(j-1));
     }
 
-    double divHeatFlux(const double* x, size_t j) const {
-        double c1 = m_tcon[j-1]*(T(x,j) - T(x,j-1));
-        double c2 = m_tcon[j]*(T(x,j+1) - T(x,j));
-        return -2.0*(c2/(z(j+1) - z(j)) - c1/(z(j) - z(j-1)))/(z(j+1) - z(j-1));
+    /**
+     * Compute the conduction term from the energy equation using a central
+     * three-point differencing scheme. This term is discretized
+     *
+     * The term to be discretized is:
+     * @f[
+     *   \frac{d}{dz}\left(\lambda \frac{dT}{dz}\right)
+     * @f]
+     *
+     * Let @f$ A = \lambda \frac{dT}{dz} @f$ for simplicity. We'll call this the inner
+     * term or inner discretization. In this situation, the inner term is evaluated
+     * using a central difference formula, but evaluated at j+1/2 and j-1/2 (halfway bet
+     * ween the grid points around point j).
+     *
+     * The value of @f$ A @f$ at point @f$ j-1/2 @f$ is estimating using a central difference
+     * formula:
+     *
+     * @f[
+     *  A_{j-1/2} = \lambda_{j-1/2} \frac{T_j - T_{j-1}}{z_j - z_{j-1}}
+     * @f]
+     *
+     * The value of @f$ A @f$ at point @f$ j+1/2 @f$ is:
+     *
+     * @f[
+     *   A_{j+1/2} = \lambda_{j+1/2} \frac{T_{j+1} - T_j}{z_{j+1} - z_j}
+     * @f]
+     *
+     * In the implementation, a vector of thermal conductivities are used which actually
+     * represents the thermal conductivity evaluated at the midpoints between grid
+     * points. In other words, the value of lambda[j] is actually
+     * @f$ \lambda_{j+1/2} @f$. The transport properties between grid points are not
+     * the average of the adjacent grid points, but rather, the state of the system at
+     * the midpoint is estimated, and then the transport properties are evaluated at
+     * that state (T,P,Y) for example.
+     *
+     * The outer discretization uses a central difference between the j+1/2 and j-1/2
+     * locations.
+     *
+     * @f[
+     *  \frac{dA}{dz} \approx \frac{A_{j+1/2} - A_{j-1/2}}{z_{j+1/2} - z_{j-1/2}}
+     * @f]
+     *
+     * Where the values of @f$ z @f$ are: @f$ z_{j+1/2} = z_{j} + 0.5*(z_{j+1} - z_j) = 0.5*(z_{j} + z_{j+1}) @f$ and
+     * @f$ z_{j-1/2} = z_j - 0.5*(z_j - z_{j-1}) = 0.5*(z_{j} + z_{j-1}) @f$. The difference between these two values is
+     * @f$ z_{j+1/2} - z_{j-1/2} = \frac{z_{j+1} - z_{j-1}}{2}
+     *
+     * Substituting these values into the central difference formula gives:
+     *
+     * @f[
+     *   \frac{d}{dz}\left(\lambda \frac{dT}{dz}\right) \approx \frac{\lambda_{j+1/2} \frac{T_{j+1} - T_j}{z_{j+1} - z_j} -
+     *     \lambda_{j-1/2} \frac{T_j - T_{j-1}}{z_j - z_{j-1}}}{\frac{z_{j+1} - z_{j-1}}{2}}
+     * @f]
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
+    double conduction(const double* x, size_t j) const {
```

Comment:
Renamed functions (in this case `divHeatFlux` -> `conduction`) need to be deprecated. Also, you introduce a lot of churn elsewhere by adding spaces, which are not introduced here (i.e. `T(x,j-1)` vs `T(x, j-1)`)

Reply:
Good catch. I noticed that in the Flow1D.cpp file there was a mash-up of spaces and not spaces through the residual functions, so I was trying to add the spaces as it seems like the space-after-comma style is used elsewhere.

Reply:
In general, we try to avoid formatting-only changes on areas that aren't being edited anyways. They tend to cause merge conflicts for little benefit.